payjoin/src/core/receive/common/mod.rs

```diff
@@ -34,7 +34,7 @@ pub struct WantsOutputs {
     payjoin_psbt: Psbt,
     params: Params,
     change_vout: usize,
-    owned_vouts: Vec<usize>,
+    pub(crate) owned_vouts: Vec<usize>,
```

Comment:
big nit, but wouldn't the cleaner pattern be a SessionEvents constructor from owned_vouts or a `pub(crate) fn owned_vouts` getter? Incongruent field accessibility is a tiny smell.